maintenance/dumpTextPass.php

```diff
@@ -269,8 +269,8 @@ function readDump( $input ) {
 		$parser = xml_parser_create( "UTF-8" );
 		xml_parser_set_option( $parser, XML_OPTION_CASE_FOLDING, false );
 
-		xml_set_element_handler( $parser, array( &$this, 'startElement' ), array( &$this, 'endElement' ) );
-		xml_set_character_data_handler( $parser, array( &$this, 'characterData' ) );
+		xml_set_element_handler( $parser, array( $this, 'startElement' ), array( $this, 'endElement' ) );
+		xml_set_character_data_handler( $parser, array( $this, 'characterData' ) );
 
 		$offset = 0; // for context extraction on error reporting
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)